torch/_dynamo/convert_frame.py

```diff
@@ -412,6 +413,12 @@ def maybe_cprofile(func):
     return func
 
 
+# Hooks for bytecode generated by Dynamo. The hook can do some
+# logging, as well as return a new code object to be used. Please
+# refer to `BytecodeHook` for the hook signature.
+output_bytecode_hooks: List[BytecodeHook] = []
```

Comment:
Put this in config.py so it can be changed via `torch.compile(options=...)`

Reply:
Can you elaborate on how this can work out? I don't know `options` in `torch.compile` can go so deeply and affect these hooks. My understanding is that `options` are some pre-defined options, and the type signature is `Optional[Dict[str, Union[str, int, bool]]]`. Since we don't provide any set of predefined hooks, I don't know how to support hooks in `torch.compile(options=...)`.

Reply:
Ah right.  I think options only effect _inductor.config, my mistake.  It is fine to leave it here.